src/main/java/teammates/ui/pagedata/InstructorFeedbackRemindParticularStudentsPageData.java

```diff
@@ -28,4 +32,24 @@ public String getFsName() {
     public FeedbackSessionResponseStatus getResponseStatus() {
         return responseStatus;
     }
+
+    /**
+     * sorts the students in the responseStatus (both who responded and with no response).
+     * The sort is by Team then by student name.
+     */
+    private void sortStudentNamesListsInResponseStatus() {
```

Comment:
If you look into `teammates.common.datatransfer.FeedbackSessionResponseStatus`, you will see that the comparison functions are already there. No need to reinvent the wheel here.